### PR TITLE
cli for new user

### PR DESCRIPTION
**Pull request by mayabrandi**
This PR cli for new user

**How to prepare for test**:
- [x] Install locally and run the command: `nipt -c <some config> load user -n <name> -r RW -e <email>`

**Expected test outcome**:
- [x] check that  then new user was added

**Review:**
- [x] code approved by @northwestwitch 
- [x] tests executed by @mayabrandi 
- [x] "Merge and deploy" approved by @mayabrandi 
Thanks for filling in who performed the code review and the test!

This [version](https://semver.org/) is a:
- [ ] **MAJOR** - when you make incompatible API changes
- [x] **MINOR** - when you add functionality in a backwards compatible manner
- [ ] **PATCH** - when you make backwards compatible bug fixes or documentation/instructions

### CODE REVIEW

File: NIPTool/commands/load/base.py [hunk 1]
```diff
@@ -10,6 +10,7 @@
```
Comment (northwestwitch) [inline NIPTool/commands/load/base.py:10]: The import lines up to line 7 of this file are not used.

**Comment by mayabrandi** [comment]
Tested according to above:

Skärmavbild 2020-09-30 kl  13 38 07
Skärmavbild 2020-09-30 kl  13 38 28

File: NIPTool/commands/load/base.py [hunk 2]
```diff
@@ -10,6 +10,7 @@
 
 # commands
 from NIPTool.commands.load.batch import batch as batch_command
```
Comment (northwestwitch) [inline NIPTool/commands/load/base.py:12]: I'm wondering: if in the future you add other batch-related command how would you name them in this file?
 Perhaps assign it a more self-explicative name, something like this:
`from NIPTool.commands.load.batch import batch as load_batch_cmd`

File: NIPTool/commands/load/base.py [hunk 3]
```diff
@@ -10,6 +10,7 @@
...
+from NIPTool.commands.load.user import user as user_command
```
Comment (northwestwitch) [inline NIPTool/commands/load/base.py:13]:
```suggestion
from NIPTool.commands.load.user import user as load_user_cmd
```

File: NIPTool/commands/load/base.py [hunk 4]
```diff
@@ -24,3 +25,4 @@ def load():
 
 
 load.add_command(batch_command)
```
Comment (northwestwitch) [inline NIPTool/commands/load/base.py:27]:
```suggestion
load.add_command(load_batch_cmd)
```

File: NIPTool/commands/load/base.py [hunk 5]
```diff
@@ -24,3 +25,4 @@ def load():
...
+load.add_command(user_command)
```
Comment (northwestwitch) [inline NIPTool/commands/load/base.py:28]:
```suggestion
load.add_command(load_user_cmd)
```

File: NIPTool/commands/load/user.py [hunk 6]
```diff
@@ -0,0 +1,23 @@
...
+from flask.cli import with_appcontext, current_app
+from NIPTool.exeptions import NIPToolError
+
+
+LOG = logging.getLogger(__name__)
+
+
+@click.command("user", short_help="load batch into db.")
+@click.option("-n", "--name", help="User name")
+@click.option("-r", "--role", help="User name")
```
Comment (northwestwitch) [inline NIPTool/commands/load/user.py:13]:
```suggestion
@click.option("-r", "--role", help="User role")
```

File: NIPTool/commands/load/user.py [hunk 7]
```diff
@@ -0,0 +1,23 @@
...
+@click.option("-e", "--email", help="User name")
```
Comment (northwestwitch) [inline NIPTool/commands/load/user.py:14]:
```suggestion
@click.option("-e", "--email", help="User email")
```

**Review by northwestwitch — CHANGES_REQUESTED** [review]
Some small things!

File: NIPTool/commands/load/user.py [hunk 8]
```diff
@@ -0,0 +1,23 @@
...
+import click
+from NIPTool.load.user import load_user
+from flask.cli import with_appcontext, current_app
+from NIPTool.exeptions import NIPToolError
+
+
+LOG = logging.getLogger(__name__)
+
+
+@click.command("user", short_help="load batch into db.")
```
Comment (northwestwitch) [inline NIPTool/commands/load/user.py:11]:
```suggestion
@click.command("user", short_help="load a user into db.")
```

File: NIPTool/commands/load/base.py (hunk 1 above)
Comment (mayabrandi) [reply NIPTool/commands/load/base.py:10]: 😣

File: NIPTool/commands/load/base.py (hunk 2 above)
Comment (mayabrandi) [reply NIPTool/commands/load/base.py]: good point

**Review by northwestwitch — APPROVED** [review]
👍 Looks good!

**Pull request merged**